### DISCUSSION TITLE
Network only works internally after installation

### DISCUSSION THREAD

**Discussion by sogeil** [ltsp/ltsp#826] [Q&A]
Hello!

I followed the installation instructions (I think), opted for chroot, and in the end my network connection isn't fully working anymore, i.e. only internally.

Ping to router works.
Ping to the www no longer works.

Possibly a problem with the name server?

I've looked at the network settings and everything seems fine.
Reboot performed.

Bodhi Linux (Ubuntu), no special features, one network card.

I haven't installed/set up a system in a while, maybe I'm rusty. I never had to deal with network setup much either.

It would be nice if someone would help.

Olaf

**Comment by alkisg** [comment]
Hi, what's the output of the following commands?

```shell
ip a
ip r
ping -c1 8.8.8.8
cat /etc/resolv.conf
host www.google.com
```

**Comment by sogeil** [comment]
`sudo service dnsmasq stop` -> Internet access works

https://ubuntuforums.org/showthread.php?t=2172787
says you should comment out the line port=0 in /etc/dnsmasq.d/ltsp.conf, but then:

sudo service dnsmasq restart
`Job for dnsmasq.service failed because the control process exited with error code.
See "systemctl status dnsmasq.service" and "journalctl -xe" for details.`


To your questions:

ip a

`1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host
       valid_lft forever preferred_lft forever
2: eno1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP group default qlen 1000
    link/ether ec:b1:d7:4b:d9:c8 brd ff:ff:ff:ff:ff:ff
    altname enp0s25
    inet 192.168.178.62/24 brd 192.168.178.255 scope global noprefixroute eno1
       valid_lft forever preferred_lft forever
`

ip r

`default via 192.168.178.1 dev eno1 proto static metric 100
192.168.178.0/24 dev eno1 proto kernel scope link src 192.168.178.62 metric 100`


ping -c1 8.8.8.8

`PING 8.8.8.8 (8.8.8.8) 56(84) Bytes Daten.
64 Bytes von 8.8.8.8: icmp_seq=1 ttl=120 Zeit=20.1 ms

--- 8.8.8.8 ping-Statistik ---
1 Pakete übertragen, 1 empfangen, 0% Paketverlust, Zeit 0ms
rtt min/avg/max/mdev = 20.092/20.092/20.092/0.000 ms`


cat /etc/resolv.conf
...
`nameserver 127.0.0.1` 


host www.google.com

`;; connection timed out; no servers could be reached`

**Comment by alkisg** [comment]
@sogeil first, try the default LTSP setup with:

```
sudo ltsp dnsmasq
```

In that case, ltsp doesn't touch your DNS settings. If DNS isn't working after that command, it's not an LTSP issue.

While if you do want to make the LTSP server a DNS server, try with:

```
sudo ltsp dnsmasq -d1
```

If none of these work for you, you may come in the chat room for live support if you wish:
http://ltsp.org/guides/chat-room/

There's no reason to modify /etc/dnsmasq.d/ltsp-dnsmasq.conf directly.

**Comment by sogeil** [comment]
$ sudo ltsp dnsmasq
Installed /usr/share/ltsp/server/dnsmasq/ltsp-dnsmasq.conf in /etc/dnsmasq.d/ltsp-dnsmasq.conf
Restarted dnsmasq

$ ping www.google.de
ping: www.google.de: Temporärer Fehler bei der Namensauflösung
(=ping: www.google.de: Temporary error in name resolution)

//

$ sudo ltsp dnsmasq -d1
Installed /usr/share/ltsp/server/dnsmasq/ltsp-dnsmasq.conf in /etc/dnsmasq.d/ltsp-dnsmasq.conf
Disabled DNSStubListener in systemd-resolved
Restarted dnsmasq

$ ping www.google.de
ping: www.google.de: Temporärer Fehler bei der Namensauflösung


$ sudo service dnsmasq stop
$ ping www.google.de
ping: www.google.de: Temporärer Fehler bei der Namensauflösung

$ sudo ltsp dnsmasq
Installed /usr/share/ltsp/server/dnsmasq/ltsp-dnsmasq.conf in /etc/dnsmasq.d/ltsp-dnsmasq.conf
Restarted dnsmasq

$ sudo service dnsmasq stop

to be able to write online here again

**Comment by alkisg** [comment]
Right, this means there's something misconfigured in your system, you may come in the chat room for live support if you wish:

http://ltsp.org/guides/chat-room/

**Comment by sogeil** [comment]
Thank you, I'll try the chat. It's difficult, I usually have to use google translator...
